<?xml version='1.0' encoding='utf-8'?>
<sdf version="1.6">
  <world name="visualize_lidar_world">
    
    <gravity>0 0 -9.8</gravity>
    <physics name="1ms" type="ignored">
      <max_step_size>0.001</max_step_size>
      <real_time_factor>1.0</real_time_factor>
    </physics>
    <plugin
            filename="libignition-gazebo-physics-system.so"
            name="ignition::gazebo::systems::Physics">
    </plugin>
    <plugin
            filename="libignition-gazebo-sensors-system.so"
            name="ignition::gazebo::systems::Sensors">
      <render_engine>ogre2</render_engine>
    </plugin>
    <plugin filename="ignition-gazebo-user-commands-system" name="ignition::gazebo::systems::UserCommands"/>
    <plugin
            filename="libignition-gazebo-scene-broadcaster-system.so"
            name="ignition::gazebo::systems::SceneBroadcaster">
    </plugin>
    <gui fullscreen="0">

      <!-- 3D scene -->
      <plugin filename="MinimalScene" name="3D View">
        <ignition-gui>
          <title>3D View</title>
          <property type="bool" key="showTitleBar">false</property>
          <property type="string" key="state">docked</property>
        </ignition-gui>

        <engine>ogre2</engine>
        <scene>scene</scene>
        <ambient_light>0.4 0.4 0.4</ambient_light>
        <background_color>0.8 0.8 0.8</background_color>
        <camera_pose>-6 0 6 0 0.5 0</camera_pose>
      </plugin>

      <!-- Plugins that add functionality to the scene -->
      <plugin filename="EntityContextMenuPlugin" name="Entity context menu">
        <ignition-gui>
          <property key="state" type="string">floating</property>
          <property key="width" type="double">5</property>
          <property key="height" type="double">5</property>
          <property key="showTitleBar" type="bool">false</property>
        </ignition-gui>
      </plugin>
      <plugin filename="GzSceneManager" name="Scene Manager">
        <ignition-gui>
          <property key="resizable" type="bool">false</property>
          <property key="width" type="double">5</property>
          <property key="height" type="double">5</property>
          <property key="state" type="string">floating</property>
          <property key="showTitleBar" type="bool">false</property>
        </ignition-gui>
      </plugin>
      <plugin filename="InteractiveViewControl" name="Interactive view control">
        <ignition-gui>
          <property key="resizable" type="bool">false</property>
          <property key="width" type="double">5</property>
          <property key="height" type="double">5</property>
          <property key="state" type="string">floating</property>
          <property key="showTitleBar" type="bool">false</property>
        </ignition-gui>
      </plugin>
      <plugin filename="CameraTracking" name="Camera Tracking">
        <ignition-gui>
          <property key="resizable" type="bool">false</property>
          <property key="width" type="double">5</property>
          <property key="height" type="double">5</property>
          <property key="state" type="string">floating</property>
          <property key="showTitleBar" type="bool">false</property>
        </ignition-gui>
      </plugin>
      <!-- World control -->
      <plugin filename="WorldControl" name="World control">
        <ignition-gui>
          <title>World control</title>
          <property type="bool" key="showTitleBar">false</property>
          <property type="bool" key="resizable">false</property>
          <property type="double" key="height">72</property>
          <property type="double" key="width">121</property>
          <property type="double" key="z">1</property>

          <property type="string" key="state">floating</property>
          <anchors target="3D View">
            <line own="left" target="left"/>
            <line own="bottom" target="bottom"/>
          </anchors>
        </ignition-gui>

        <play_pause>true</play_pause>
        <step>true</step>
        <start_paused>true</start_paused>
        <use_event>true</use_event>

      </plugin>

      <!-- World statistics -->
      <plugin filename="WorldStats" name="World stats">
        <ignition-gui>
          <title>World stats</title>
          <property type="bool" key="showTitleBar">false</property>
          <property type="bool" key="resizable">false</property>
          <property type="double" key="height">110</property>
          <property type="double" key="width">290</property>
          <property type="double" key="z">1</property>

          <property type="string" key="state">floating</property>
          <anchors target="3D View">
            <line own="right" target="right"/>
            <line own="bottom" target="bottom"/>
          </anchors>
        </ignition-gui>

        <sim_time>true</sim_time>
        <real_time>true</real_time>
        <real_time_factor>true</real_time_factor>
        <iterations>true</iterations>
      </plugin>


      <!-- Inspector -->
      <plugin filename="ComponentInspector" name="Component inspector">
        <ignition-gui>
          <property type="string" key="state">docked</property>
        </ignition-gui>
      </plugin>

      <!-- Entity tree -->
      <plugin filename="EntityTree" name="Entity tree">
        <ignition-gui>
          <property type="string" key="state">docked</property>
        </ignition-gui>
      </plugin>
    </gui>

        <model name="aws_robomaker_warehouse_ShelfF_01_001">
        <include>
            <uri>model://aws_robomaker_warehouse_ShelfF_01</uri>
        </include>
        <pose frame="">-5.795143 -0.956635 0 0 0 0</pose>
	</model>

        <model name="aws_robomaker_warehouse_WallB_01_001">
        <include>
            <uri>model://aws_robomaker_warehouse_WallB_01</uri>
        </include>
        <pose frame="">0.0 0.0 0 0 0 0</pose>
	</model>

        <model name="aws_robomaker_warehouse_ShelfE_01_001">
        <include>
            <uri>model://aws_robomaker_warehouse_ShelfE_01</uri>
        </include>
        <pose frame="">4.73156 0.57943 0 0 0 0</pose>
	</model>

        <model name="aws_robomaker_warehouse_ShelfE_01_002">
        <include>
            <uri>model://aws_robomaker_warehouse_ShelfE_01</uri>
        </include>
        <pose frame="">4.73156 -4.827049 0 0 0 0</pose>
	</model>

        <model name="aws_robomaker_warehouse_ShelfE_01_003">
        <include>
            <uri>model://aws_robomaker_warehouse_ShelfE_01</uri>
        </include>
        <pose frame="">4.73156 -8.6651 0 0 0 0</pose>
	</model>

        <model name="aws_robomaker_warehouse_ShelfD_01_001">
        <include>
            <uri>model://aws_robomaker_warehouse_ShelfD_01</uri>
        </include>
        <pose frame="">4.73156 -1.242668 0 0 0 0</pose>
	</model>

        <model name="aws_robomaker_warehouse_ShelfD_01_002">
        <include>
            <uri>model://aws_robomaker_warehouse_ShelfD_01</uri>
        </include>
        <pose frame="">4.73156 -3.038551 0 0 0 0</pose>
	</model>

        <model name="aws_robomaker_warehouse_ShelfD_01_003">
        <include>
            <uri>model://aws_robomaker_warehouse_ShelfD_01</uri>
        </include>
        <pose frame="">4.73156 -6.750542 0 0 0 0</pose>
	</model>

        <!--model name="aws_robomaker_warehouse_DeskC_01_001">
        <include>
            <uri>model://aws_robomaker_warehouse_DeskC_01</uri>
        </include>
        <pose frame="">-0.061684 6.135864 0 0 0 0</pose>
	</model>

        <model name="aws_robomaker_warehouse_DeskC_01_002">
        <include>
            <uri>model://aws_robomaker_warehouse_DeskC_01</uri>
        </include>
        <pose frame="">-0.061684 3.039 0 0 0 0</pose>
	</model>

        <model name="aws_robomaker_warehouse_DeskC_01_003">
        <include>
            <uri>model://aws_robomaker_warehouse_DeskC_01</uri>
        </include>
        <pose frame="">-0.061684 -6.6493 0 0 0 0</pose>
	</model-->

        <model name="aws_robomaker_warehouse_GroundB_01_001">
        <include>
            <uri>model://aws_robomaker_warehouse_GroundB_01</uri>
        </include>
        <pose frame="">0.0 0.0 -0.090092 0 0 0</pose>
	</model>

        <model name="aws_robomaker_warehouse_Lamp_01_005">
        <include>
            <uri>model://aws_robomaker_warehouse_Lamp_01</uri>
        </include>
        <pose frame="">0 0 -4 0 0 0</pose>
	</model>



        <model name="aws_robomaker_warehouse_Bucket_01_020">
        <include>
            <uri>model://aws_robomaker_warehouse_Bucket_01</uri>
        </include>
        <pose frame="">0.433449 9.631706 0 0 0 -1.563161</pose>
	</model>

        <model name="aws_robomaker_warehouse_Bucket_01_021">
        <include>
            <uri>model://aws_robomaker_warehouse_Bucket_01</uri>
        </include>
        <pose frame="">-1.8321 -6.3752 0 0 0 -1.563161</pose>
	</model>

        <model name="aws_robomaker_warehouse_Bucket_01_022">
        <include>
            <uri>model://aws_robomaker_warehouse_Bucket_01</uri>
        </include>
        <pose frame="">0.433449 8.59 0 0 0 -1.563161</pose>
	</model>

  <model name='aws_robomaker_warehouse_ClutteringA_01_016'>
        <include>
            <uri>model://aws_robomaker_warehouse_ClutteringA_01</uri>
        </include>
        <pose frame=''>5.708138 8.616844 -0.017477 0 0 0</pose>
	</model>

      <model name='aws_robomaker_warehouse_ClutteringA_01_017'>
        <include>
            <uri>model://aws_robomaker_warehouse_ClutteringA_01</uri>
        </include>
        <pose frame=''>3.408638 8.616844 -0.017477 0 0 0</pose>
	</model>

      <model name='aws_robomaker_warehouse_ClutteringA_01_018'>
        <include>
            <uri>model://aws_robomaker_warehouse_ClutteringA_01</uri>
        </include>
        <pose frame=''>-1.491287 5.222435 -0.017477 0 0 -1.583185</pose>
	</model>


     <model name="aws_robomaker_warehouse_ClutteringC_01_027">
        <include>
            <uri>model://aws_robomaker_warehouse_ClutteringC_01</uri>
        </include>
        <pose frame="">3.324959 3.822449 -0.012064 0 0 1.563871</pose>
	</model>

     <model name="aws_robomaker_warehouse_ClutteringC_01_028">
        <include>
            <uri>model://aws_robomaker_warehouse_ClutteringC_01</uri>
        </include>
        <pose frame="">5.54171 3.816475 -0.015663 0 0 -1.583191</pose>
	</model>

     <model name="aws_robomaker_warehouse_ClutteringC_01_029">
        <include>
            <uri>model://aws_robomaker_warehouse_ClutteringC_01</uri>
        </include>
        <pose frame="">5.384239 6.137154 0 0 0 3.150000</pose>
	</model>

     <model name="aws_robomaker_warehouse_ClutteringC_01_030">
        <include>
            <uri>model://aws_robomaker_warehouse_ClutteringC_01</uri>
        </include>
        <pose frame="">3.236 6.137154 0 0 0 3.150000</pose>
	</model>

     <model name="aws_robomaker_warehouse_ClutteringC_01_031">
        <include>
            <uri>model://aws_robomaker_warehouse_ClutteringC_01</uri>
        </include>
        <pose frame="">-1.573677 2.301994 -0.015663 0 0 -3.133191</pose>
	</model>

     <model name="aws_robomaker_warehouse_ClutteringC_01_032">
        <include>
            <uri>model://aws_robomaker_warehouse_ClutteringC_01</uri>
        </include>
        <pose frame="">-1.2196 9.407 -0.015663 0 0 1.563871</pose>
	</model>

     <model name='aws_robomaker_warehouse_ClutteringD_01_005'>
        <include>
            <uri>model://aws_robomaker_warehouse_ClutteringD_01</uri>
        </include>
        <pose frame=''>-1.634682 -7.811813 -0.319559 0 0 0</pose>
	</model>

      <model name='aws_robomaker_warehouse_TrashCanC_01_002'>
        <include>
            <uri>model://aws_robomaker_warehouse_TrashCanC_01</uri>
        </include>
        <pose frame=''>-1.592441 7.715420 0 0 0 0</pose>
	</model>


      <model name='aws_robomaker_warehouse_PalletJackB_01_001'>
        <include>
            <uri>model://aws_robomaker_warehouse_PalletJackB_01</uri>
        </include>
        <pose frame=''>-0.276098 -9.481944 0.023266 0 0 0</pose>
	</model>

  </world>
</sdf>
